app/hotels/src/filter/Delimiter.js

```diff
@@ -0,0 +1,45 @@
+// @flow
```

Comment:
We already have these components, they can be imported as
`import { SeparatorFullWidth, SeparatorTrimmed } from '@kiwicom/mobile-navigation'`